src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -20,56 +20,53 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-import kubernetesClusterExplorer from 'images/kubernetes_screenshot-full_cluster-explorer.gif'
+import kubernetesClusterExplorer from 'images/kubernetes_screenshot-crop_cluster-explorer.webp'
 
 import kubernetesDashboard from 'images/kubernetes_screenshot-full_dashboard.webp'
 
 import kubernetesEvents from 'images/kubernetes_screenshot-full_events.gif'
 
-Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. The New Relic Kubernetes monitoring integration gives you visibility into your Kubernetes clusters and workloads in minutes, whether your clusters are hosted on-premises or in the cloud.
 
-Thanks to our [cluster explorer](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer), you can cut through layers of complexity to see how your cluster is performing, from the heights of the control plane down to applications running on a single pod.
+Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. The New Relic Kubernetes monitoring integration gives you a quick and easy way to see what's going on with your Kubernetes clusters and workloads, whether they're hosted on-premises or in the cloud.
 
-<img
-title="New Relic - Kubernetes Cluster Explorer"
-alt="New Relic - Kubernetes Cluster Explorer"
-src={kubernetesClusterExplorer}
-/>
+Once you've got the [Kubernetes integration up and running](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/), you can start monitoring all the services running on Kubernetes in one place with to our [Kubernetes out-of-the-box experiences](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer). It'll show you everything from the heights of the control plane down to applications running on a single pod, so you can get to the heart of how your cluster is performing.
```

Comment:
```suggestion
Once you've got the [Kubernetes integration up and running](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/), you can start monitoring all the services running on Kubernetes in one place with our [Kubernetes out-of-the-box experiences](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer). It'll show you everything from the heights of the control plane down to applications running on a single pod, so you can get to the heart of how your cluster is performing.
```

Reply:
"From the heights" feels a little awkward. What you're trying to get at is that you can see the entire system from a holistic perspective as well as with a focus on the minute. Can you rewrite this to focus on the different views and perspectives New Relic offers without the image (even though don't get me wrong, I love a good image!)

Reply:
Thank you, Sammie. I'm going to replace the last sentence with this:
`It shows you everything from the top of the control plane down to the applications running on a single pod, so you can get to the heart of how your cluster works.`
I can't add the different views and remove the image for now, but it's a good idea and I'll do it soon.